data/mods/ssb/moves.ts

```diff
@@ -84,7 +82,6 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		desc: "If this move is successful and the user has not fainted, the effects of Leech Seed and binding moves end for the user, and all hazards are removed from the user's side of the field. Raises the user's Speed by 1 stage.",
 		shortDesc: "Free user from hazards/bind/Leech Seed; +1 Spe.",
 		name: "Skystriker",
-		isNonstandard: "Custom",
```

Comment:
im glad noone gets metronome in ssb4 or hoo boy

Reply:
ya that is why i removed it